pkgs/by-name/ce/celeste-classic/package.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, stdenvNoCC
+, fetchzip
+, autoPatchelfHook
+, SDL2
+, practiceMod ? false
+}:
+
+let
+  directory = if practiceMod then "CELESTE*Practice*" else "CELESTE";
+  srcbin = if practiceMod then "celeste_practice_mod" else "celeste";
+  outbin = if practiceMod then "celeste-classic-pm" else "celeste-classic";
+in
+stdenvNoCC.mkDerivation {
+  pname = outbin;
+  version = "1.0";
+
+  src = fetchzip {
+    url = "https://www.speedrun.com/static/resource/174ye.zip";
+    hash = "sha256-GANHqKB0N905QJOLaePKWkUuPl9UlL1iqvkMMvw/CC8=";
+  };
```

Pin it to what looks to a specific version (so this doesn't break or rely on the cache if there's ever a new version) and note where it came from.
```suggestion
	# From https://www.speedrun.com/celestep8/resources
  src = fetchzip {
    url = "https://www.speedrun.com/static/resource/174ye.zip?v=f3dc98f";
    hash = "sha256-GANHqKB0N905QJOLaePKWkUuPl9UlL1iqvkMMvw/CC8=";
  };
```